lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -324,8 +324,34 @@ export async function findPr({
   prTitle,
   state = 'all',
   refreshCache,
+  includeOtherAuthors,
 }: FindPRConfig): Promise<Pr | null> {
   logger.debug(`findPr(${branchName}, "${prTitle!}", "${state}")`);
+
+  if (includeOtherAuthors) {
+    // PR might have been created by anyone, so don't use the cached Renovate PR list
+    const searchParams: Record<string, string> = {
+      state: 'OPEN',
+    };
+    searchParams['direction'] = 'outgoing';
+    searchParams['at'] = `refs/heads/${branchName}`;
+
+    const query = getQueryString(searchParams);
+    const prs = await utils.accumulateValues(
```

Comment:
I seemed repetitive to write the api call here, since we already have a fn which does the same thing. Both ways ie. calling api directly from there or calling the accumulator we get the same result array and accumulator loop breaks after one run, so not much of a difference ig.